sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/DocumentAnalysisTestEnvironment.cs

```diff
@@ -15,82 +15,74 @@ public class DocumentAnalysisTestEnvironment : TestEnvironment
     {
         private const string SanitizedSasUrl = "https://sanitized.blob.core.windows.net";
 
-        public DocumentAnalysisTestEnvironment()
-        {
-        }
-
-        /// <summary>The name of the environment variable from which the Form Recognizer resource's endpoint will be extracted for the live tests.</summary>
-        internal const string EndpointEnvironmentVariableName = "FORM_RECOGNIZER_ENDPOINT";
```

Comment:
These constants are used literally in only one place in this class, so removing them to simplify code.

Reply:
A lot of these variables are shared between FRTestEnvironment and DocAnalysisTestEnvironment, should we share the code here?

Reply:
Most of them have the same name but are not the same variable. For example, `MultipageBlobContainerSasUrl` pulls data from env var `MULTIPAGE_BLOB_CONTAINER_SAS_URL` in DocAnalysisTestEnvironment, but uses env var `MULTIPAGE_BLOB_CONTAINER_SAS_URL_V2` in FRTestEnvironment. The files used to train our models in service version 2.0 are not compatible with 3.0, and vice-versa.

Our test classes either use one environment or the other (never both), so I think it makes sense to keep them separated.